src/app/server/Server.h

```diff
@@ -298,6 +301,7 @@ class Server
 
     ServerTransportMgr mTransports;
     SessionManager mSessions;
+    SimpleSessionResumptionStorage mSessionResumptionStorage;
```

Comment:
Suggest injecting this, and making this ownership based `CommonCaseDeviceServerInitParams` for the moment. This will allow manufacturers to replace the implementation and will allow usecases where both Server and Controller are supported to *inject the same cache*

Reply:
Done, make SessionResumptionStorage injectable.